WalletWasabi/WabiSabi/Backend/PostRequests/ArenaRequestHandler.cs

```diff
@@ -32,6 +32,12 @@ public ArenaRequestHandler(WabiSabiConfig config, Prison prison, Arena arena, IR
 
 		public async Task<InputRegistrationResponse> RegisterInputAsync(InputRegistrationRequest request, CancellationToken cancellationToken)
 		{
+#if !DEBUG
```

Comment:
i guess this is because there's a lot of `Main` sprinkled around the unit tests, but maybe it's safer to avoid mainnet for now in debug builds too?